plugin/rename.py

```diff
@@ -132,20 +134,17 @@ def is_visible(
     def input(self, args: dict) -> sublime_plugin.TextInputHandler | None:
         if "new_name" in args:
             # Defer to "run" and trigger rename.
-            return None
-        prepare_provider_session = self.best_session("renameProvider.prepareProvider")
+            return
```

Comment:
I'd prefer this to explicitly return `None`, unless `None` is the only possible return value of the function (which is not the case here).

See [PEP8](https://peps.python.org/pep-0008/#programming-recommendations):

> Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function (if reachable):